### ISSUE TITLE
mat-datepicker:  Accessibility Issue while running fast Pass

### ISSUE THREAD

**Issue by deepakparamesh** [angular/components#20531]
#### Reproduction

On running the 'Fast Pass' test in **[Accessibility Insights for Web](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni)**, we get 'aria-allowed-attr' issue in V8.2.3

![image](https://user-images.githubusercontent.com/12880320/92731719-14078280-f393-11ea-980d-ac716379b971.png)

![image](https://user-images.githubusercontent.com/12880320/92731779-25508f00-f393-11ea-9596-545653852488.png)

Steps to reproduce:
1. Install  **[Accessibility Insights for Web](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni)** plugin in either Google chrome or Chromium Edge
2. Run the Fast Pass test by opening a date picker.
 
 
#### Expected Behavior

aria-selected="false" should be removed from <td>

#### Actual Behavior

aria-selected="false" is present in the <td> tag.

#### Environment

- Angular: 8.2.14
- CDK/Material: 8.2.3
- Browser(s): Edge Chromium, Google Chrome
- Operating System (e.g. Windows, macOS, Ubuntu): Windows, MacOs, Ubuntu

**Comment by crisbeto** [comment]
Closing, because [according to the W3C](https://www.w3.org/TR/wai-aria-1.1/#aria-selected), `false` is a valid value for `aria-selected`. It sounds like something that shouldn't be flagged by the extension in the first place. For what it's worth, I tried running the extension over the same page and it only flagged some places where labels were missing:

![Automated_checks_-_Accessibility_Insights_for_Web__2020-09-13_13-00-45](https://user-images.githubusercontent.com/4450522/93016402-34b51f80-f5c1-11ea-8e69-4595e01773c7.png)
